content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md

```diff
@@ -8,22 +8,44 @@ aliases:
 keywords: [platform-setup,kubernetes,minikube]
 ---
 
-Follow these instructions to prepare Minikube for Istio.
+Follow these instructions to prepare Minikube for Istio installation with sufficient
+resources to run Istio and some basic applications.
 
 1. To run Istio locally, install the latest version of
    [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.1 or
    later**.
 
-1. Select a
-   [VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
-   and substitute `your_vm_driver_choice` below with the installed virtual
-   machine (VM) driver. To install Istio control plane components and addons,
-   as well as other applications,
-   we recommend starting Minikube with 8192 `MB` of memory and 4 `CPUs`:
+1. Select a [Minikube VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
```

Comment:
Do we really want to dwell on this vm driver stuff here? It's just part of installing minikube. I myself use virtualbox which is the default, so I don't even need to set the `--vm-driver` and this whole part will just be noise.

Reply:
I don't think we want to dwell on it, but we do want to make sure people don't come back and say "what you said doesn't work".  For example, I use VMWare's mac hypervisor...